packages/onboarding/src/step-container-v2/components/TopBar/style.scss

```diff
@@ -69,5 +69,12 @@
 		margin-left: auto;
 		font-size: 0.875rem;
 		line-height: 1;
+
+		a,
+		button {
+			&.components-button:not(.is-destructive).is-link {
```

Comment:
Requires high specificity to override https://github.com/Automattic/wp-calypso/blob/ad4df861dc68724a37438d9d5e8e7be063c65e35/client/assets/stylesheets/_wp-components-overrides.scss#L42